_pages/attributes.md

```diff
@@ -1,22 +1,26 @@
 ---
-title: Available Attributes
+title: User attributes
 ---
 
-# Available attributes
+# User attributes
 
-Here are the available attributes that can be requested when a user is validated.
+Along with the user's UUID, here are the default attributes that can be requested when a user is validated.
 
-| Attribute      | Default |
-|----------------|---------|
-| first_name     | Yes     |
-| middle_name    | Yes     |
-| last_name      | Yes     |
-| address1       | Yes     |
-| address2       | Yes     |
-| city           | Yes     |
-| state          | Yes     |
-| zipcode        | Yes     |
-| dob            | Yes     |
-| ssn            | Yes     |
-| phone          | Yes     |
-| email          | No      |
+For LoA 1 accounts:
+
+- Email
+
+For LoA 3 accounts:
```

Comment:
I liked the table structure, what do you think about a structure like this?

| Attribute | LOA1 | LOA3 |
| ---- | --- | --- |
| first_name | | ✓ |
| email | ✓ | ✓ |

etc